### ISSUE TITLE
Create multiple category axises with value axises

### ISSUE THREAD

**Issue by manoj016** [amcharts/amcharts4#295]
How to create multiple category axises with values axises in LineSeries XY chart?

**Comment by martynasma** [comment]
You can add any number of axes of any type, including multiple category axes.

Here's a very quick example of a simple chart with multiple category axes:
https://codepen.io/team/amcharts/pen/9fe386a8e0046927df308c5224a55fb1

Please note, if the number of data items do not match, you can also add `data` directly to series and axis.

For more info on axis, check this guide:
https://www.amcharts.com/docs/v4/concepts/axes/

**Comment by manoj016** [comment]
Thank you very much,

I am giving wrong data format, so it is not rendering properly, like

`
data = [
   {
        ts1: 1533435,
         value1: 13243
   },
   {
        ts2: 32454,
        value2: 3435
   }
]
`

**Comment by manoj016** [comment]
If I want to show one category axises for both `t1` and `t2`. Can I do this?

**Comment by martynasma** [comment]
I'm afraid, I'm not entirely sure what you mean.

You want to have a single category axis and use both `ts1` and `ts2` fields as its `dataFields.category`?

I'm afraid that's not possible. You will need to modify your data to unify them into a single field.

If you can't modify your data at its source, you can do so using an adapter, e.g.:

```
chart.adapter.add("data", function(data) {
  var newdata = [];
  for (var i = 0; i < data.length; i++) {
    if (data[i].ts1 !== undefined) {
      newdata.push({
        ts: data[i].ts1,
        value1: data[i].value1
      });
    }
   else {
      newdata.push({
        ts: data[i].ts2,
        value1: data[i].value2
      });
    }
  }
  return newdata;
});
```

**Comment by martynasma** [comment]
In that case, why don't you want to plot your data using `DateAxis` instead?

**Comment by manoj016** [comment]
I want to show every x axis data in the chart but in my case it shows only one because all the data are produced in one day.

**Comment by manoj016** [comment]
Can you give me an example of like this like some seconds of difference data in X axis using DateAxis

**Comment by martynasma** [comment]
OK, there you go:
https://codepen.io/team/amcharts/pen/EeZvBo?editors=0010

It's a chart that has two series, with non-matching timestamps, plotted on the same DateAxis.

Hope you find this useful.

**Comment by manoj016** [comment]
yeah thank you its a nice example by the way.

I tried this and its working fine.

But in my case, it shows `invalide date` and the x axis is not rendering.
<img width="277" alt="screen shot 2018-09-03 at 3 26 04 pm" src="https://user-images.githubusercontent.com/41284266/44980591-f0342980-af8d-11e8-8135-aa219fc3a871.png">

**Comment by martynasma** [comment]
Would you be able to create a live codepen?

**Comment by manoj016** [comment]
Thank you @martynasma for your helps.
I want a help regarding the tooltip in this chart that the tootip will be shown with the x axis value and y axis value, I tried it using `{dateX} {valueY}` it shows the date for `dateX`. Basically I want only time in hh:mm:ss format in the tooltip. Can you help me to sort it out.

Thank you,

**Comment by martynasma** [comment]
Yes, you can specify format directly in the `tooltipText`:

```
series.tooltipText = "{dateX.formatDate('HH:mm:ss')}: {valueY}";
```

https://www.amcharts.com/docs/v4/concepts/formatters/formatting-date-time/#Overriding_format

**Comment by manoj016** [comment]
When I use this `series.tooltipText = "{dateX.formatDate('HH:mm:ss')}: {valueY}";`

It shows invalid date in the tooltip of series.

Example is here.
https://codepen.io/manoj016/pen/jePmKz?editors=0010

**Comment by martynasma** [comment]
Good catch. It's a bug.

We've just fixed in dev. Will release updated version before end of week.

Will ping you here once that happens.

Your code will work then.

**Comment by manoj016** [comment]
ok, Thank you very much @martynasma

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.56](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.56).

## [4.0.0-beta.56] - 2018-10-12

### Added
- New `Sprite` adapter: `"populateString"`. Applied to strings after `Sprite` replaces data binders with real values, but before any formatting by `Label` kicks in.
- New `Label` property: [`ignoreFormatting`](https://www.amcharts.com/docs/v4/reference/label/#ignoreFormatting_property). If set to `true` will treat formatting blocks in square brackes (e.g. `"[red bold]"`) as regular text and will display them as such.

### Changed
- Default big, small, and byte suffixes in `NumberFormatter` are now translatable via locale files.
- The suffix for thousands "kilo", was changed to lowercase "k", as per [standards](https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes).
- On `XYChart`, default for [`mouseWheelBehavior`](https://www.amcharts.com/docs/v4/reference/xychart/#mouseWheelBehavior_property) changed to `"none"` (was `"zoomX"`).

### Fixed
- Improved performance! All charts types are now noticably, some drastically are faster. Some memory leakages fixed as well.
- Now `MapPolygonSeries` will not inherit chart's global `geodata` if it has own `geodataSource` set.
- `DateFormatter` was not parsing all AM/PM string dates correctly.
- `DateFormatter` was not parsing `MMM` (short month names like "Jan", "Feb") propertly.
- In-line date formatter was not working correctly in some cases, e.g. `"{dateX.formatDate()}"`.
- Setting `chart.mouseWheelBehavior = "none"` did not release wheel scrolling of the web page when hovering over chart.
- `DataSource`'s' `"done"` event was being called if parsing of loaded data failed. Not anymore. `"done"` is called only on successful load. To catch any load (including one that ended in http or parse error) use `"ended"`.
- `DataSource`'s' `reloadFrequency` was broken if error occurred during one of the loads. This has been fixed. Even if error occurrs, loader will try to reload data in the next reload slot.
- JSON: `heatRules` did not work properly with their target set to a list template, e.g. `mapPolygons.template`.
- Auto-calculated `DateAxis.baseInterval` was not being carried over to the `XYChartScrollbar`.
- `DurationAxis` scale sometimes was dropping to negative values, even there were no negative values in data.
- `PieChart` and `Legend` were not working properly with `chart.zoom()` or `legend.zoom()` methods.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
